### DISCUSSION TITLE
Runtime Logs - how to add custom meta data?

### DISCUSSION THREAD

**Discussion by AlonMiz** [vercel/community#6453] [Feature Question] [answered]
### Question

a main feature I would expect from the "runtime logs" is to have custom fields.
for example, I would like to add, a user id/ org id
another ability would be to extract context from the request/log. I would like to get the `requestId` and send it to my other logs.

is extending the log is possible while using next.js?
thanks

**Comment by Jooliashi** [comment, marked as answer]
Hello,

at the moment this is not possible; if you want the `requestId` to be sent to your other logs, it sounds more like a functionality in log drain (we do send requestId in log drains)
